WordPress/Classes/ApplicationToken/ApplicationTokenListView.swift

```diff
@@ -58,14 +66,13 @@ class ApplicationTokenListViewModel: ObservableObject {
     @Published
     private(set) var applicationTokens: [ApplicationTokenItem]
 
-    private let dataProvider: ApplicationTokenListDataProvider!
+    let dataProvider: ApplicationTokenListDataProvider!
```

Comment:
Does it need a force unwrap?

Reply:
Good catch! Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/23834/commits/db61f33a68cd8833d4fbf2edb4af814a2409b082